packages/fscomponents/src/components/TabbedContainer.tsx

```diff
@@ -42,33 +38,23 @@ const styles = StyleSheet.create({
   }
 });
 
-export class TabbedContainer extends Component<TabbedContainerProps, TabbedContainerState> {
-  state: TabbedContainerState;
-
-  constructor(props: TabbedContainerProps) {
-    super(props);
+export const TabbedContainer = (props: TabbedContainerProps): JSX.Element => {
```

Comment:
Please use the `TabbedContainer: React.FunctionComponent<TabbedContainerProps>` typing notation